Week2/homework/maartjes-work.js

```diff
@@ -43,19 +43,28 @@ const tuesday = [
 ];
 
 const maartjesTasks = monday.concat(tuesday);
+
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const mappedTasks = tasks.map(task => task.duration / 60);
+  const filteredTasks = mappedTasks.filter(task => task >= 2);
+  const multiplyTasks = filteredTasks.map(task => task * hourlyRate);
+  const roundedTasks = multiplyTasks.map(task => task);
+  const sumTasks = roundedTasks.reduce((a, b) => a + b);
```

Comment:
You are summing up the earnings here, so the name `sumTasks` is not entirely wrong but nor very informative either. A better name would be `totalEarnings`.